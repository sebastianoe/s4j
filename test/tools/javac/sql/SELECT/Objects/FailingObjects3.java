/*
 * @test  (c) SO
 * @ignore
 * @summary Fail due to missing id (note: currently disabled as an explicit id isn't necessarily required)
 * @compile businessobjects/Product.java
 * @compile businessobjects/Storage.java
 * @compile/fail/ref=FailingObjects3.out FailingObjects3.java -sqlrepo /Users/sebastian/ma/jdk7u-langtools/test/tools/javac/sql/repo.json
 */

import businessobjects.Product;
import businessobjects.Storage;

public class FailingObjects3 {
    public static void main(String[] args) {
    	Product p = SQL[SELECT * FROM storage WHERE id = 1];
    }
}