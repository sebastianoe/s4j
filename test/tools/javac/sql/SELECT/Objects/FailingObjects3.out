error: The mapping did not include id-projection assignments for the sub-objects: root
1 error